packages/xo-web/src/xo-app/menu/index.js

```diff
@@ -88,6 +91,11 @@ export default class Menu extends Component {
     isEmpty
   )
 
+  _getNoNotifications = createSelector(
+    () => this.props.notifications,
+    notifications => every(notifications, n => !n.read)
```

Comment:
This doesn't look right. `every(notifications, { read: true })`?

Reply:
We check if we haven't notifications so `every(notifications, { read: false })`

Reply:
"Not having any notifications" means that you read all of them. So *every* notification is *read*.